src/citrine/informatics/design_spaces/data_source_design_space.py

```diff
@@ -25,6 +25,7 @@ class DataSourceDesignSpace(Resource['DataSourceDesignSpace'], DesignSpace, AIRe
     """
 
     _response_key = None
+    _resource_type = ResourceTypeEnum.MODULE
```

Comment:
Because `Resource` is extended at the level of individual modules, this line needs to be duplicated for each one. Ugh